spec/controllers/admin/tracks_controller_spec.rb

```diff
@@ -253,4 +258,204 @@
       end
     end
   end
+
+  describe 'PATCH #restart' do
+    before :each do
+      self_organized_track.state = 'canceled'
+      self_organized_track.save!
+      patch :restart, conference_id: conference.short_title, id: self_organized_track.short_name
+      self_organized_track.reload
+    end
+
+    it 'assigns the correct track' do
+      expect(assigns(:track)).to eq self_organized_track
+    end
+
+    it 'shows message in flash notice' do
+      expect(flash[:notice]).to eq 'Review for My awesome track started!'
+    end
+
+    it 'changes the track\'s state to new' do
+      expect(self_organized_track.state).to eq 'new'
+    end
+  end
+
+  describe 'PATCH #to_accept' do
+    before :each do
+      patch :to_accept, conference_id: conference.short_title, id: self_organized_track.short_name
+      self_organized_track.reload
+    end
+
+    it 'assigns the correct track' do
+      expect(assigns(:track)).to eq self_organized_track
+    end
+
+    it 'shows message in flash notice' do
+      expect(flash[:notice]).to eq 'Track My awesome track marked as a possible acceptance!'
+    end
+
+    it 'changes the track\'s state to to_accept' do
+      expect(self_organized_track.state).to eq 'to_accept'
+    end
+  end
+
+  describe 'PATCH #accept' do
+    shared_examples 'fails to accept' do |start_date, end_date, room|
+      before :each do
```

Comment:
I'd re-write this part of your shared examples, and move these to the individuals examples.
The true/false values don't make a lot of sense, unless you scroll to see what they refer to. And the way you are assigning values to some of the attributes inside the shared_example is not making it easy to read/understand. 

You should be defining what you need defined and then call the shared_example, ie the common things you want to run for multiple scenarios.

You can create the self_organized_track record every time with the attributes you want, and always use that variable inside your shared example.